charts/redpanda/templates/_configmap.tpl

```diff
@@ -165,8 +165,8 @@ redpanda.yaml: |
         port: {{ $listener.port }}
   {{- end }}
 {{- end }}
-    admin_api_tls:
```

Comment:
To be clear, nothing will break if this is empty nor if this is left out?

Reply:
It seems this is the case, which is surprising to me as it could result in an empty section in the yaml. I'm not sure why disabling TLS works without the change in this PR.